modules/ptp-configuring-linuxptp-services-as-ha-bc-for-dual-nic.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ptp-configuring-linuxptp-services-as-ha-bc-for-dual-nic_{context}"]
+= Configuring linuxptp as highly available boundary clock for dual NIC Intel e810 hardware
+
+You can configure `linuxptp` services (`ptp4l`, `phc2sys`) as a highly available boundary clock (T-BC) for dual Intel e810 Salem channel dual NIC hardware.
```

Comment:
same here: r High Availability of the system clock when there are multiple time sources on different NICs. SystemClock --> phc2sys